hackathon_site/hackathon_site/__init__.py

```diff
@@ -0,0 +1,2 @@
+MIN_MEMBERS = 2
+MAX_MEMBERS = 4
```

Comment:
There might be some confusion here, could you put these constants in `hackathon_site/settings/__init__.py` we don't put anything in hackathon_site/__init__.py because our setup goes in the settings folder

in `hackathon_site/settings/__init__.py` could you put the constants here:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/20618460/173109365-1f84c9fa-8da6-4e03-aeba-ee9ab2a81eaa.png">

you can also put a small comment above it e.g. `# Team requirements` to follow our current format

You'll have to change your imports everywhere too. Wherever you use these constants you should be importing settings from django (not the hackathon_site folder) like this `from django.conf import settings` and then you should be able to do `settings.MIN_MEMBERS` or `settings.MAX_MEMBERS` to get the value of these constants.